jiezi/templates/index.html

```diff
@@ -1,82 +1,95 @@
-<!DOCTYPE html>
-<html lang='en'>
-<head>
-  <meta charset='utf-8'>
-  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-	<meta name='viewport' content='width=device-width, initial-scale=1'>
+{% extends 'base.html' %}
+{% load static %}
+{% block css %}
+    <link rel='stylesheet' type="text/css" href='{% static "styles/dashboard.css" %}'>
+{% endblock %}
+{% block content %}
+    <h1>Welcome back, {{ user.first_name }}!</h1>
+    <a href="{% url "manage_library" %}"class='button button-primary' href='/vocab.html'>Manage Vocab List</a>
```

Comment:
here use single quotation mark for outside and double for inside, or the other way

Reply:
this is yjm's code style, but I can fix that